package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "fxa-geodb",
+  "version": "0.0.2",
+  "private": true,
+  "description": "Firefox AccountsGeoDB Repo for Geolocation based services",
+  "main": "src/fxa-geodb.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "cover": "istanbul cover _mocha",
+    "lint": "grunt lint",
+    "postinstall": "node scripts/max-loader.js",
+    "setup": "npm install && bower install",
```

Comment:
Good catch @pdehaan We dont, this was leftover from me incorrecly assuming this was on client side.